### PR TITLE
Release notes for Docker Desktop 3.2.1

### PR DESCRIPTION
**Pull request by StefanScherer**
Signed-off-by: Stefan Scherer <stefan.scherer@docker.com>

Release notes for the Docker Desktop 3.2.1 patch release.

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -16,10 +16,23 @@ This page contains information about the new features, improvements, known issue
...
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Upgrades
+
+- [Docker Engine 20.10.5](https://docs.docker.com/engine/release-notes/#20105)
+
+### Bug fixes and minor changes
+
+- Fixed a problem starting Docker Desktop 3.2.0 during a canary release. Fixes [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406)
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:30]: I don't think we've talked about canary releases publicly. Suggest "... 3.2.0 after an update"

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/release-notes.md:30]: Fixed an issue that sometimes caused Docker Desktop to fail to start after upgrading to version 3.2.0. Fixes [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406).

**Comment by usha-mandya** [comment]
Suggested a minor change. Looks good otherwise. Thanks for the PR.

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (stephen-turner) [reply docker-for-mac/release-notes.md:30]: @usha-mandya Do you prefer "updating" or "upgrading"? I thought we tried to reserve "upgrade" for getting a paid plan, but I'm not sure we're very careful about it.

Comment (usha-mandya) [reply docker-for-mac/release-notes.md:30]: That's a good point. We've used both 'update' and 'upgrade' previously. Agree that we should use 'update' in this context. This is also consistent with the UI (Check for Updates). Thanks for flagging.

Comment (usha-mandya) [reply docker-for-mac/release-notes.md:30]: I've 'updated' my previous comment ;)

File: docker-for-mac/release-notes.md [hunk 2]
```diff
@@ -52,7 +65,7 @@ This page contains information about the new features, improvements, known issue
 
 ### Known issue
 
-Docker Desktop can sometimes fail to start after upgrading to version 3.2.0. We are working on a fix for this issue. If you are experiencing this issue, we recommend that you uninstall 3.2.0 and manually install [Docker Desktop 3.1.0](#docker-desktop-310). See [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406).
+Docker Desktop can sometimes fail to start after upgrading to version 3.2.0. We are working on a fix for this issue. If you are experiencing this issue, we recommend that you uninstall 3.2.0 and manually install [Docker Desktop 3.2.1](#docker-desktop-321). See [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406).
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:68]:
```suggestion
Docker Desktop can sometimes fail to start after updating to version 3.2.0. We are working on a fix for this issue. If you are experiencing this issue, we recommend that you uninstall 3.2.0 and manually install [Docker Desktop 3.2.1](#docker-desktop-321). See [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406).
```

Comment (StefanScherer) [reply docker-for-mac/release-notes.md]: Thanks, I'll 'update' this :-)

**Review by usha-mandya — APPROVED** [review]
LGTM

**Review by stephen-turner — APPROVED** [review]
Yup

**Pull request merged**